README.md

```diff
@@ -57,22 +57,48 @@ podman run -v $PWD:/app --platform linux/amd64 -it \
 
 ### Tests
```

Comment:
you can also use the legder VsCode extension to easily perform your tests as described here : 
https://github.com/LedgerHQ/app-boilerplate#test